app/main.cjsx

```diff
@@ -16,8 +16,7 @@ style = require '../css/main.styl'
 store = configureStore()
 auth.listen('change', () => store.dispatch(emptySubjectQueue()));
 apiClient.type('subject_sets').listen('add-or-remove', () => store.dispatch(emptySubjectQueue()));
-sugarClient.on('experiment', (message) => store.dispatch(notify(message)));
-sugarClient.on('subject-queue', (message) => store.dispatch(injectSubjects(message)));
+sugarClient.on('experiment', (message) => store.dispatch(handleInterventionMessage(message)));
```

Comment:
Where is `handleInterventionMessage` defined?

Reply:
blargh - thanks for catching.